metriq_gym/cli.py

```diff
@@ -14,20 +13,8 @@ def parse_arguments() -> argparse.Namespace:
     parser = argparse.ArgumentParser(description="Metriq-Gym benchmarking CLI")
     subparsers = parser.add_subparsers(dest="action", required=True, help="Action to perform")
 
-    # The `jobs_file` argument is common to both dispatch and poll subcommands.
```

Comment:
👍  with removing this argument; that level of freedom to the users only make our life of maintainers more complicated.

Reply:
Yep, I think this was a good suggestion. Glad we're doing that here.